buildSrc/src/main/groovy/org/apache/beam/gradle/VendorJavaPlugin.groovy

```diff
@@ -131,15 +131,15 @@ artifactId=${project.name}
         inputs.files project.configurations.shadow.artifacts.files
         doLast {
           project.configurations.shadow.artifacts.files.each {
-            FileTree exposedClasses = project.zipTree(it).matching {
+            FileTree unexpectedlyExposedClasses = project.zipTree(it).matching {
               include "**/*.class"
               exclude "org/apache/beam/vendor/**"
               // BEAM-5919: Exclude paths for Java 9 multi-release jars.
-              exclude "**/module-info.class"
```

Comment:
A similar change is needed in:
https://github.com/apache/beam/blob/1133f7b5ae5d10120563e267d7c65f669ccc2bf1/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L970

Reply:
Done